### DISCUSSION TITLE
Duda sobre enunciado

### DISCUSSION THREAD

**Discussion by KaterinaMariescurren** [frlp-utn-ingsoft/catedra#102] [Q&A] [answered]
Hola, buenas tardes. Tenemos duda con el punto 4. a), que dice lo siguiente:
4. Distribución Aislada y Consistente: 
a. Cada estudiante deberá compartir la infraestructura Docker que han creado mediante un commit y push en su repositorio de GitHub.
No comprendemos a que se hace referencia con compartir la infrestructura Docker.

**Comment by ivanduva** [comment, marked as answer]
@rodriguezemautn
